docs/tutorials/start-a-private-network/4customchain.md

```diff
@@ -0,0 +1,115 @@
+
+The portion of the file we're interested in is the Aura authorities (used for creating blocks) and the GRANDPA authorities (used for finalizing blocks). That section looks like this
+```json
+"aura": {
+  "authorities": [
+    [
+      "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
+      1
+    ],
+    [
+      "5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty",
+      1
+    ]
+  ]
+},
+"grandpa": {
+  "authorities": [
+    [
+      "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
+      1
+    ],
+    [
+      "5GoNkf6WdbxCFnPdAnYYQyCjAKPJgLNxXwPjwTh6DGg6gN3E",
+      1
+    ]
+  ]
+},
+
+```
+
+All we need to do is change the authority addresses listed (currently Alice and Bob) to our own addresses that we generated in the previous step. The sr25519 addresses go in the `aura` section, and the ed25519 addresses in the `grandpa` section. You may add as many validators as you like. For additional context, you can see how polkadot uses these keys and others in the [Polkadot wiki](https://wiki.polkadot.network/docs/en/learn-keys#session-keys).
+
+> A single person should do these steps and share the resulting file with their fellow validators.
+>
+> Because Rust -> Wasm builds aren't "reproducible", each person will get a slightly different Wasm blob which will break consensus if each participant generates the file themselves.
+
+Once the chainspec is prepared, convert it to a "raw" chainspec. The raw chainspec contains all the same information, but it contains the encoded storage keys that the node will use to reference the data in its local storage. Distributing a raw spec ensures that each node will store the data at the proper storage keys.
+
+```bash
+./target/release/node-template build-spec --chain customSpec.json --raw > customSpecRaw.json
+```
+
+Finally share the `customSpecRaw.json` with your all the other validators in the network.
+
+## First Participant starts a Bootnode
+
+You've completed all the necessary prep work and you're now ready to launch your chain. This process is very similar to when you launched a chain earlier as Alice and Bob. It's important to start with a clean base path, so if you plan to use the same path that you've used previously, please delete all contents from that directory.
+
+The first participant can launch her node with
+```bash
+./target/release/node-template \
+  --chain ./customSpecRaw.json \
+  --port 30333 \
+  --ws-port 9944 \
+  --rpc-port 9933 \
+  --telemetry-url ws://telemetry.polkadot.io:1024 \
+  --validator
+  --name MyNode
+```
+Here are some differences from when we launched as Alice.
+* I've omitted the `--base-path` flag, so Substrate will use a default location. You're still free to specify your own base path, and you must do so if multiple nodes will run on the same physical machine.
```

Comment:
The thought here is that the user would set up another local network with 2 machines? Why not have them specify a base path again? It seems they would have to.

In either case, I think it makes sense to teach them about `purge-chain` if you have been teaching them to delete the folder up until now.